src/m365/aad/commands.ts

```diff
@@ -28,8 +29,7 @@ export default {
   O365GROUP_REPORT_ACTIVITYDETAIL: `${prefix} o365group report activitydetail`,
   O365GROUP_REPORT_ACTIVITYFILECOUNTS: `${prefix} o365group report activityfilecounts`,
   O365GROUP_REPORT_ACTIVITYGROUPCOUNTS: `${prefix} o365group report activitygroupcounts`,
-  O365GROUP_REPORT_ACTIVITYSTORAGE: `${prefix} o365group report activitystorage`,
-  O365GROUP_RESTORE: `${prefix} o365group restore`,
```

Comment:
We should keep the old name for backwards compatibility

Reply:
Sure. I'll keep the old name